### PR TITLE
Create Shuffle_An_Array.java

### PR DESCRIPTION
**Pull request by yatharthshahjpr**
Code for Shuffling an array using Java ! :)

**Changes:**
[Code in java has been made for the purpose of randomizing an array]

### CODE REVIEW

File: code/randomized-algorithms/shuffle_an_array/Shuffle_An_Array.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+              array[i] = array[j];
+              array[j] = temp;
+            }
+		System.out.println(Arrays.toString(array));
+	}
+  
+   public static void main(String[]args)
+  {
+          int[] A = {1,2,3,4,5,6,7,8};
+          int n = A.length();
```
Comment (AdiChat) [inline code/randomized-algorithms/shuffle_an_array/Shuffle_An_Array.java]: length is a property of an array in `Java` but you are referring to an undefined function. You, probably, want this statement: ` int n = A.length;`

**Comment by AdiChat** [comment]
Kindly make the above change and this will be good to be merged 😍

File: code/randomized-algorithms/shuffle_an_array/Shuffle_An_Array.java (hunk 1 above)
Comment (AdiChat) [reply code/randomized-algorithms/shuffle_an_array/Shuffle_An_Array.java]: I fixed it 👍

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Comment by yatharthshahjpr** [comment]
Thank you !  :)

File: code/randomized-algorithms/shuffle_an_array/Shuffle_An_Array.java (hunk 1 above)
Comment (yatharthshahjpr) [reply code/randomized-algorithms/shuffle_an_array/Shuffle_An_Array.java]: Thank you ! 👍

**Pull request merged**